indexer/categories_holder.hpp

```diff
@@ -113,6 +113,7 @@ class CategoriesHolder
       {"vi", 36},
       {"zh-Hans", kSimplifiedChineseCode},
       {"zh-Hant", kTraditionalChineseCode},
+      {"eu", 39},
```

Comment:
Let's sort them by name. I know, it's annoying, we'll refactor it in the future.

Reply:
Done